daffodil-test-ibm1/src/test/scala/org/apache/daffodil/TresysTests.scala

```diff
@@ -139,7 +143,7 @@ class TresysTests {
 
   @Test def test_nested_group_refs1() { runnerNG.runOneTest("nestedGroupRefs1") }
 
-  @Test def test_AF000() { runnerAF.runOneTest("AF000") }
+  @Test def test_AF000() { runnerAF.trace.runOneTest("AF000") }
```

Comment:
Remove the trace call here. I believe this will ennable debugging for all runnerAF tests, since trace mutates the state?